src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
                 return new Session($content);
             },
+            'users/signup' => function (array $content) {
+                return new User($content);
+            },
+            'users/forgot-password' => function (array $content) {
```

Comment:
What do you mean ? that's not password reset but that's password forgot. there is no redirect. It's email with link for reset password